index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio</title>
+
+
+</head>
+
+
+<body>
+    <!--header-->
+    <!--nav menu-->
+    <div class="nav-bar ">
+        <div class="logo">
+             <a href="#" class="mylogo">CHESTER</a> </div>
+        <button><img src=" /images/Union.png " alt=" " class="hamburger "> </button>
+        <!--closing nav-->
+    </div>
+
+    <article>
+
+
+        <h1>I m rotshidzwa Glad to see
+            <br>you!</h1>
+
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2>LET’S CONNECT</h2>
```

Comment:
- [ ] **Required:** You are required to have this in a `button` or `a` tag and style accordingly.

![image](https://user-images.githubusercontent.com/27709832/159553064-f998d99f-1bcc-46e7-a8a5-2ab45e4069fa.png)